tuannguyensn2001/edu-rails
# [PEAC-881] API Login

**Jira Ticket:**
- https://employmenthero.atlassian.net/browse/PEAC-881

**Summary:**
- User login, using jwt to generate and verify

**Actions:**
- Find user in database by email
- Compare request password and user password
- Send job Notify
- Generate access token and refresh token
- Return response

**Testing plan**
- Setup local env (https://github.com/tuannguyensn2001/edu-rails)
- Setup postman
- Start server
- Send request to: http://localhost:3000/api/v1/auth/login, body:  {
    "email": "tuannguyensn2001a@gmail.com",
    "password": "123456"
}
- Expect response has access token and refresh token
